### DISCUSSION TITLE
Next Releases

### DISCUSSION THREAD

**Discussion by ans-4175** [begundal-ug/penyegaran_tl#15] [General]
Diskusi disini apapun untuk next releases bertahap

**Comment by ans-4175** [comment]
Setelah MVP, baiknya ke fase mana?
Menurutku ke Ready to Share https://github.com/begundal-ug/penyegaran_tl/milestone/2 @anwari666 
Gimana?
bagian yang leaderboard soalnya agak ribet di awal, mending dapat feedback dulu aja baru ke fitur yang ribet?

**Reply by ans-4175** [reply]
MVP tinggal bagian https://github.com/begundal-ug/penyegaran_tl/issues/9

**Reply by anwari666** [reply]
Baru ngeh, di GH ada section buat diskusi sendiri. Beda ya ama Issues....

Anyway, si milestone Ready to Share itu udah OK. Aku paling nambahin untuk share single page + show original tweet nya ya. Harusnya bisa dikejar minggu ini sih hehe. Yang penting rilis & koleksi data di GA dulu aja. Lihat preferensi & aktivitas orang2 kayak gimana...

**Reply by anwari666** [reply]
Oiya gimana kalau aplikasinya pakai bahasa indonesia dulu aja? Termasuk bagian About --> Tentang.

**Reply by ans-4175** [reply]
Hmm, boleh sih biar kelokalan kl gitu..

**Reply by anwari666** [reply]
Gadis2nya lokal semua pak :v

**Reply by ans-4175** [reply]
> Baru ngeh, di GH ada section buat diskusi sendiri. Beda ya ama Issues....
> 
> Anyway, si milestone Ready to Share itu udah OK. Aku paling nambahin untuk share single page + show original tweet nya ya. Harusnya bisa dikejar minggu ini sih hehe. Yang penting rilis & koleksi data di GA dulu aja. Lihat preferensi & aktivitas orang2 kayak gimana...

Btw, bagaimana cara kita reach out orang-orang buat memakai ini?
Share di forum? Share di twitter fess anonim? Share di tempat/channel mana yang orang akan banyak ngecek dan persona target kita

**Reply by ans-4175** [reply]
Sekarang kita fokus buat nyebarin @anwari666 dan pasang GA buat ngelihat interaksinya, mastiin di interaksinya itu.
Sambil mencari channel buat nyebari dan dapatin feedbacknya. Kira-kira kemana aja ya?

**Reply by anwari666** [reply]
Sip. Yang jelas dicoba ke admin penyegaran sih... Selain itu paling via forum2, milis random, IRC, komunitas terbatas, gitu2? Aku asa ga siap kl kita merilis di sosmed mainstream xD

**Reply by anwari666** [reply]
btw apakah kita perlu mengamankan handle @penyegaran_ml di twitter?

**Reply by ans-4175** [reply]
menurutku sih ga perlu ya, ga kepakai juga kan sama kita?

Admin penyegaran ini jarang aktif dan gabisa di-dm euy.
Mending masuk ke semacam forum, twitter base?, menfess?, akun-akun viral yang mirip personanya?

Sosmed mainstream itu apa?

**Reply by anwari666** [reply]
menfess teh apa?

Sosmed mainstream itu ya twitter, ig, fb..

**Comment by ans-4175** [comment]
Issues #5 dan #6 sementara aku closed/stale dulu @anwari666 karena dari interaksi dll belum masuk dan bisa jadi obsolete.

**Reply by anwari666** [reply]
setuju pak
